main.cpp

```diff
@@ -283,23 +307,23 @@ struct CarDealership
     int carSold = 50;
     int staff = 23;
     int carServiced = 40;
-    int invBought = 15;
-    float carSale(float price);
-    void salesmen();
-    int invator(int invatory);
+    int invatoryBought = 15;
+    float sellCars(float price);
+    void hireSalesmen();
+    int buyInvatory(int invatory);
 };
```

Comment:
`inventory`, not `invatory`

It is important to make your code as easy to understand as possible.  that starts with choosing excellent variable and function names, and also spelling them correctly. 
- [ ] revise